Bio/SeqIO/InsdcIO.py

```diff
@@ -1085,6 +1085,13 @@ def write_record(self, record):
             taxonomy = "."
         self._write_multi_line("", taxonomy)
 
+        if "db_source" in record.annotations:
+            # Hack around the issue of BioSQL loading a list for the db_source
```

Comment:
Pragmatic, and less risky than touching the BioSQL code.

Reply:
I thought so. My assumption was that this is being run as an internal utility, whereas the Loader for BioSQL was likely a public (user-facing) API. Addressing the fact that it's loading this annotation as a list would be a potentially breaking change. This does rely on the idea that there are never more than 1 actual `db_source` annotations, but I wasn't able to find any sort of specification for the file format that included a reference to `db_source`.